app/src/bcsc-theme/features/account/AccountExpiredScreen.tsx

```diff
@@ -0,0 +1,74 @@
+import { useFactoryReset } from '@/bcsc-theme/api/hooks/useFactoryReset'
+import { AppBannerSection, BCSCBanner } from '@/bcsc-theme/components/AppBanner'
+import { CardButton } from '@/bcsc-theme/components/CardButton'
+import { useAccount } from '@/bcsc-theme/contexts/BCSCAccountContext'
+import { BCSCMainStackParams, BCSCScreens } from '@/bcsc-theme/types/navigators'
+import { useTheme } from '@bifold/core'
+import { StackNavigationProp } from '@react-navigation/stack'
+import { useTranslation } from 'react-i18next'
+import { ScrollView, StyleSheet, View } from 'react-native'
+import { SafeAreaView } from 'react-native-safe-area-context'
+import HomeHeader from '../home/components/HomeHeader'
+
+interface AccountExpiredScreenProps {
+  navigation: StackNavigationProp<BCSCMainStackParams, BCSCScreens.AccountExpired>
+}
+
+/**
+ * Renders the Account Expired screen, informing users that their account has expired and providing options to renew or remove the account.
+ *
+ * @returns {*} {JSX.Element} The AccountExpiredScreen component.
+ */
+export const AccountExpiredScreen = ({ navigation }: AccountExpiredScreenProps): JSX.Element => {
+  const { t } = useTranslation()
+  const { Spacing } = useTheme()
+  const factoryReset = useFactoryReset()
+  const account = useAccount()
+
+  const styles = StyleSheet.create({
+    container: {
+      flex: 1,
+    },
+    scrollContainer: {
+      padding: Spacing.md,
+      gap: Spacing.lg,
+    },
+    actionsContainer: {
+      gap: Spacing.md,
+    },
+  })
+
+  return (
+    <SafeAreaView edges={['bottom', 'left', 'right']} style={styles.container}>
+      <AppBannerSection
```

Comment:
nit: on dismissal of the app banner the info bellow moves up, not sure if that is fine or should stay centered.

<img width="270" height="606" alt="Image" src="https://github.com/user-attachments/assets/b17dde47-8de8-40fe-ab66-f658cc3bc9f1" />